ROADMAP.md

```diff
@@ -11,28 +11,50 @@ features. Reach us through Issues, Slack and / or Google Group!
 # Planned Features
 The following features are considered for the near future:
 
-* **Windows Kubernetes Node**
-Support Windows Kubernetes Nodes. At the moment Antrea supports only Linux Nodes.
+* **Windows support improvements**
+Antrea supports Windows K8s Node since version 0.7.0. However, a few features
+like network flow export are not supported for Windows Node yet. We will
+continue to add more features for Windows, and improve Antrea Agent and OVS
+installation on Windows Nodes.
 
 * **IPv6 Pod network**
 Support IPv6 and IPv4/IPv6 dual-stack for Pod network. Right now Antrea supports
 only IPv4.
 
-* **Network and NetworkPolicy troubleshooting**
-We want to focus on troubleshooting, and will add more mechanisms to simplify
-troubleshooting Antrea network and the NetworkPolicy implementation.
+* **Antrea NetworkPolicy**
+Antrea starts to support [Antrea native NetworkPolicy](/docs/network-policy.md)
+besides K8s NetworkPolicy since version 0.8.0. We already support
+ClusterNetworkPolicy and tiered NetworkPolicy, and will continue to add more
+NetworkPolicy extensions, like traffic logging, policy statistics, policy
+realization status, more matching criteria and actions.
 
-* **Prometheus integration**
-Export metrics to [Prometheus](https://prometheus.io) from both Antrea Controller and Agents.
+* **Network diagnostics and observability**
+Network diagnostics and observability is one area we want to focus on. Antrea
+already implements some useful features on this front, including Octant UI
+plugin, CLI, Traceflow, network flow export and visualization, Prometheus
+metrics, OVS flow dumping and tracing. We will continue to enhance existing
+features and add new features to help diagnosing K8s network and NetworkPolicy
+implementation, and to provide good visibility of Antrea network.
```

Comment:
suggestion: good visibility into the Antrea network

Reply:
Done